storage/remote/write_handler_test.go

```diff
@@ -292,6 +294,12 @@ type mockHistogram struct {
 	fh *histogram.FloatHistogram
 }
 
+func requireEqual(t *testing.T, expected, actual interface{}, msgAndArgs ...interface{}) {
```

Comment:
can we give this a more descriptive name and/or comment, just so it's really obvious that this is needed so we can compare unexported fields on the mock types?

otherwise I would just assume at first glance that we have a wrapper of `testutil.RequireEqual` that's not adding any additional testing functionality